### PR TITLE
Update V1 Metadata Device Swagger Schema

### PR DESCRIPTION
**Pull request by AnthonyMBonafide**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://wiki.edgexfoundry.org/display/FA/Contributor%27s+Guide
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?
The Swagger representation of Device is not in sync with the model that is used in go-mod-core-contracts

Issue Number: #2532 

## What is the new behavior?
The swagger matches the Device data structure

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Are there any specific instructions or things that should be known prior to reviewing?
To verify these changes copy the YAML from this PR into the Swagger editor at https://editor.swagger.io/ and verify the `POST /v1/device` request body structure matches the device structure in go-mod-core-contracts

## Other information
N\A

### CODE REVIEW

File: api/openapi/v1/core-metadata.yaml [hunk 1]
```diff
@@ -2143,11 +2156,14 @@ components:
...
-        addressable:
-          $ref: '#/components/schemas/device_addressable'
         adminState:
           title: adminState
           type: string
+        autoevents:
+          type: array
+          items:
+            title: autoevent
+            $ref: '#/components/schemas/autoevent'
```
Comment (brandonforster) [inline api/openapi/v1/core-metadata.yaml:2166]: Please add a corresponding autoevent schema.

Comment (brandonforster) [reply api/openapi/v1/core-metadata.yaml:2166]: ...unless I'm misunderstanding something and it can reference a schema inside core-contracts

Comment (AnthonyMBonafide) [reply api/openapi/v1/core-metadata.yaml:2166]: This is referencing the object `autoevent` on line 2119.

**Comment by AnthonyMBonafide** [comment]
Closing and re-opening to fix the "stuck" DCO CI Job

**Pull request merged**